marcelo_zoccoler/entry_user_interf4/Readme.md

```diff
@@ -327,11 +345,15 @@ or a *wheel* using
 python setup.py bdist_wheel
 ```
 
-The difference between both is, in short, that the source distribution provides, as the name suggests, the source files that can be downloaded and have to be compiled upon installation of the package. Python wheels, on the contrary, come pre-built, which leads to faster installation. This can be convenient for large packages. For a more in-depth on the advantages of either strategy, see [this blog](https://medium.com/ochrona/understanding-python-package-distribution-types-25d53308a9a). For simple projects, there is no damage in simply providing both. 
+The difference between both is, in short, that the source distribution provides, as the name suggests, the source files that can be downloaded and have to be compiled upon installation of the package. Python wheels, on the contrary, come pre-built, which leads to faster installation. This can be convenient for large packages. For a more in-depth on the advantages of either strategy, see [this blog](https://medium.com/ochrona/understanding-python-package-distribution-types-25d53308a9a). For simple projects, there is no damage in simply providing both, like ```python setup.py sdist bdist_wheel ```
+
+A new folder should appear in your repository, called `dist`, with a couple files inside. Before uploading to PyPi, check if this folder contains only files related to the latest version. If files named after previous versions are present, you can manually delete them.
+
+### Upload to PyPi
 
 Lastly, upload your package to PyPi with `twine` (use `pip install twine` if not present already):
 ```
-twine upload dist/*
+python -m twine upload --repository pypi dist/*
```

Comment:
Just a quick question: what does --repository do?

Reply:
According to the [documentation](https://twine.readthedocs.io/en/stable/#commands), it points to the destination repository, in this case, pypi. Since this is also the default option, I will remove that for simplicity and let it as it was.